sync-config.sh

```diff
@@ -11,14 +11,15 @@ else
 	exit
 fi
 
-if [ ! -f /config/keys/private.key ]; then
+if [ ! -f /config/private.key ]; then
 	echo "No private key found, unable to decrypt"
 fi
-if [ ! -f /config/keys/public.key ]; then
+if [ ! -f /config/public.key ]; then
 	echo "No public key found, unable to verify signatures"
 fi
-gpg2 --import /config/keys/private.key 2>/dev/null
-gpg2 --import /config/keys/public.key 2>/dev/null
-rm -rf /config/keys
+gpg2 --import /config/private.key 2>/dev/null
```

Comment:
It's weird that we check if these files exist, then try to load+delete them anyways (why u no `else`?)